fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVPlanUtil.java

```diff
@@ -150,4 +173,117 @@ public static Set<TableIf> getBaseTableFromQuery(String querySql, ConnectContext
             }
         }
     }
+
+    public static List<ColumnDefinition> generateColumnsBySql(String querySql, ConnectContext ctx, String partitionCol,
```

Comment:
add java doc style's comment to explain what this function want to do, the input paramters and the return value